packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/Action/elements.ts

```diff
@@ -0,0 +1,121 @@
+import styled, { keyframes, css } from 'styled-components';
+import { Link } from 'react-router-dom';
+import BaseMoreInfoIcon from 'react-icons/lib/md/arrow-drop-down';
+import Tooltip from '@codesandbox/common/lib/components/Tooltip';
+import { OptionProps } from './types';
+
+interface LightThemeProps {
+  theme: { light: boolean };
+}
+const blinkAnimation = keyframes`
+  0% {
+  // @ts-ignore;
+    color: ${({ theme: { light } }: LightThemeProps) =>
+    light ? 'rgba(0, 0, 0, 0.7)' : 'rgba(255, 255, 255, 0.7)'}
+    };
+
+    50 % {
+      color: rgba(255, 255, 255, 1);
+    }
+
+100 % {
+  color: ${({ theme: { light } }: LightThemeProps) =>
+    light ? 'rgba(0, 0, 0, 0.7)' : 'rgba(255, 255, 255, 0.7)'}
+  };
+`;
+
+const styles = ({
+  blink,
+  hideBottomHighlight,
+  highlight,
+  theme,
+}: OptionProps) =>
+  css`
+    ${blink &&
+    css`
+        animation: ${blinkAnimation} 1s infinite;
+        font-weight: 600;
+      `};
+    display: flex !important;
+    transition: 0.3s ease all;
+    flex- direction: row;
+    align- items: center;
+    vertical- align: middle;
+    font- size: 0.875rem;
+    line- height: 1;
+    height: 100 %;
+    color: ${theme.light ? css`#636363` : css`rgba(255, 255, 255, 0.7)`};
+    cursor: pointer;
+    box- sizing: inherit;
+    border- bottom: 2px solid transparent;
```

Comment:
There's a space here, I'm not sure if it's relevant.

Reply:
Oh I think it's relevant for highlights.

Reply:
I think those spaces between `box- sizing` and `border- bottom` are errors, probably want to inspect the element in the browser to see if an invalid rule is declared to make sure this isn't intentional. (I don't believe it is)

Reply:
Yes, I think they are used for hover highlights.